appveyor.yml

```
@@ -22,6 +22,17 @@ init:
 - SET JAVA_HOME=C:\Program Files\Java\jdk19
 - SET PATH=%JAVA_HOME%\bin;%PATH%
 
+install:
+  # If a newer build is queued for the same PR, cancel this one.
+  # The AppVeyor 'rollout builds' option is supposed to serve the same
+  # purpose, but it is problematic because it tends to cancel builds pushed
+  # directly to main/master instead of just PR builds (or the converse).
+  # credits: JuliaLang developers.
+  - ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_BUILD_NUMBER -ne ((Invoke-RestMethod `
```

yamllint

[warning] wrong indentation: expected 0 but found 2

(indentation)